cmd/kubelet/app/server_unsupported.go

```diff
@@ -0,0 +1,25 @@
+// +build !linux
+
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package app
+
+import "errors"
+
+func watchForLockfileContention(path string, done chan struct{}) error {
+	return errors.New("kubelet unsupported in this build")
```

Comment:
We support development on OSX to some extent. So we want integration tests
to function on OSX.

On Wed, May 18, 2016 at 3:57 PM, Derek Parker <notifications@github.com>
wrote:

> In cmd/kubelet/app/server_unsupported.go
> <https://github.com/kubernetes/kubernetes/pull/25596#discussion_r63797541>
> :
>
> > +
> > +    http://www.apache.org/licenses/LICENSE-2.0
> > +
> > +Unless required by applicable law or agreed to in writing, software
> > +distributed under the License is distributed on an "AS IS" BASIS,
> > +WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
> > +See the License for the specific language governing permissions and
> > +limitations under the License.
> > +*/
> > +
> > +package app
> > +
> > +import "errors"
> > +
> > +func watchForLockfileContention(path string, done chan struct{}) error {
> > +	return errors.New("kubelet unsupported in this build")
>
> Yes, if this flag was supplied, but the kubelet doesn't actually work
> correctly on OSX, correct?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/25596/files/46ddb9c0dd62d6ad34e9dcf8e3c8ff5d62fac4d2#r63797541>
>